pipeline.ttl

```diff
@@ -1,25 +1,25 @@
-@prefix : <http://emmo.info/oteio/pipeline#> .
+@prefix : <http://onto-ns.com/meta/oteapi/pipeline#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix xml: <http://www.w3.org/XML/1998/namespace> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix dcterms: <http://purl.org/dc/terms/> .
-@base <http://emmo.info/oteio/pipeline> .
+@base <http://onto-ns.com/meta/oteapi/pipeline> .
 
-<http://emmo.info/oteio/pipeline> rdf:type owl:Ontology ;
-                                           owl:versionIRI <http://emmo.info/0.3.0/oteio/pipeline> ;
+<http://onto-ns.com/meta/oteapi/pipeline> rdf:type owl:Ontology ;
+                                           owl:versionIRI <http://onto-ns.com/meta/oteapi/1/pipeline> ;
```

Comment:
It is more standard that the version comes between the general namespace and before the root component of the ontology.